backend/src/getInfo/midleware/ValidationMidleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+import { validateOrReject } from 'class-validator';
+
+export function ValidationMidleware(dto: new () => any) {
+    return async (req: Request, res: Response, next: Function) => {
+        try {
+            const newDto = new dto;
+            const properties = Object.keys(req.body);
+            properties.forEach((val) => {
+                newDto[val] = req.body[val];
+            })
+            await validateOrReject(newDto);
+            next();
+        } catch(errors: any) {
```

Comment:
you can't specify here anything except any, but it's better to specify below with specific class-validator type like:
`const errs = (errors as ValidationError[]);`